demos/smart_classroom_demo_gapi/include/kernel_packages.hpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include <opencv2/gapi/gkernel.hpp>
+
+namespace kp {
+    cv::gapi::GKernelPackage gallery_kernels();
+    cv::gapi::GKernelPackage video_process_kernels();
+    cv::gapi::GKernelPackage top_k_kernels();
```

Comment:
Why do we need three different ones? Are kernels in these packages conflicting with each other?